### PR TITLE
Create 2024-01-11

### PR DESCRIPTION
**Pull request by mashaeva-va**
Release Note for CHAPI

### CODE REVIEW

File: content/health/clinical-fhir-api/release-notes/2024-01-08 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+We have updated MedicationRequest resource to include unreleased medications data.
+• No changes are made to the existing general MedicationRequest calls, no unreleased medications are returned.
```
Comment (joeabraun) [inline content/health/clinical-fhir-api/release-notes/2024-01-08:2]:
```suggestion
• No changes are made to the existing general MedicationRequest calls, and no unreleased medications are returned. 
```

File: content/health/clinical-fhir-api/release-notes/2024-01-08 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+We have updated MedicationRequest resource to include unreleased medications data.
+• No changes are made to the existing general MedicationRequest calls, no unreleased medications are returned. 
+• When there is a search for a specific MedicationRequest using *identifier*, the view is changed to search for the medications only for that patient and include the unreleased medications.
```
Comment (joeabraun) [inline content/health/clinical-fhir-api/release-notes/2024-01-08:3]:
```suggestion
• When a search request uses a patient identifier, a VistA medication order ID, and the VistA site number where the order was placed, results will only show that patient's medications, including their unreleased medications.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Made a couple of changes.

File: content/health/clinical-fhir-api/release-notes/2024-01-08 [hunk 3]
```diff
@@ -0,0 +1,3 @@
+We have updated MedicationRequest resource to include unreleased medications data.
```
Comment (joeabraun) [inline content/health/clinical-fhir-api/release-notes/2024-01-08:1]:
```suggestion
We have updated the MedicationRequest resource to include unreleased medications data.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]

**Review by joeabraun — APPROVED** [review]
UX approves!

**Review by jlsexton — DISMISSED** [review]

**Review by jlsexton — APPROVED** [review]

**Review by jlsexton — APPROVED** [review]

**Review by joeabraun — APPROVED** [review]
UX approves.

**Pull request merged**